os/arch/Kconfig

```diff
@@ -764,6 +860,16 @@ config BOOT_COPYTORAM
 endchoice
 
 menu "Boot Memory Configuration"
+
+config RAM_START
```

Comment:
Ram_start and ram_size is being used in some architecture files

Reply:
Yes, original Nuttx uses, but we don't. We introduced new configs, RAM_KREGIONx_START and SIZE instead of them. I think we should use ours.

Reply:
https://github.com/Samsung/TizenRT/commit/7ea39b88baee4fbcedd95475f7c6269833c49127 is the commit I said.

Reply:
I think it will be used for paging in order to map physical to virtual address space. Currently, we are allocating the heap according to RAM_KREGIONx_START only.

https://github.com/Tirtha-Samsung/TizenRT/blob/d478abe7907cfbd3f0e3bea4c6f697b179d280ea/os/arch/arm/src/armv7-a/arm_allocateheap.c#L114